src/App.tsx

```diff
@@ -1,11 +1,22 @@
+import { BrowserRouter } from "react-router-dom"
+import { CardImage } from "./components/CardImage"
+import { CardColor } from "./components/CardColor"
 import { Footer } from "./components/Footer"
-import { Router } from "./routing/Router"
+import { routes } from "./routing/routes"
 
 export const App = () => {
+  const { department, field } = routes
```

Comment:
Akurat routes to jest jedna z takich zmiennych których się nie rozbija, lepiej jest mieć w kodzie `routes.field()` bo od razu wiemy że są to scieżki